### DISCUSSION TITLE
Welcome to computer-vision-course Discussions!

### DISCUSSION THREAD

**Discussion by johko** [huggingface/computer-vision-course#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sarthak247** [comment]
Greetings,
My name is Sarthak (Sam for short). I am a Masters student in Australia currently working on Large Language Models and for this course I am working on two tasks - Basic CV Tasks and GANs part along with the rest of my team members. 
You can find me on Discord (@hamster.uwu) for more. Looking forward to getting to know more people as well apart from my current team members.
Cheers :v:

**Comment by Vishnunkumar** [comment]
Hi everyone,

I am Vishnu currently working as Machine Learning Engineer in one of the reputed banks in India. I am working on contributing to notebooks predominantly comprising multimodal and NLP implementations. Looking forward to contribute and connect with everyone and grow bigger and better. 😄

**Comment by bellabf** [comment]
Hello everyone 👋!
 I am Isabella. 
I started working on computer vision related tasks about a year and a half ago, and I still remember how different computer vision felt compared to tabular data. So when I saw that I could help people learn more about the field, I felt called to contribute.  I am currently working on the Fundamentals of Computer Vision and the Image and Imaging chapters ✍️.

**Comment by ATaylorAerospace** [comment]
Hello Everyone,✌️

My name is Art and I am a Cloud ERP and Analytics Architect and a Generative AI Engineer based in Southern California. I have strong research interest in Multimodal Computer Vision especially as it relates to both Health Care and Aerospace IOT (Satellite and other Space applications) applications.

I am assisting on the Computer Vison course with chapter reviews and where I can with content contribution.

You can find me on github @ (https://github.com/ATaylorAerospace)

**Comment by wo0jun** [comment]
Hi everyone! 🤗

I am Woojun, a master's student in AI in South Korea. My research interests are in multimodal, especially video. 
I'm part of a team called Hugging Face KREW(Korean Researchers and Engineers' Wing) in South Korea and contribute to Hugging Face in various ways. 
Right now, I'm assisting on the video part of the Computer Vision course.

**Comment by Anindyadeep** [comment]
Hello everyone, :hugs: 

I am Anindya, I am a final yeat undergrad student. I am also working as an ML Engineer at @premAI-io. I am currently working on Language Models and alignment. I love contributing to open source. In this community, I am currently working on two chapters:

1. Fine-tuning ViTs for object detections.
2. Introduction to ViTs.

Would love to connect with you all. Reach out to me [here](https://bento.me/anindyadeep)

**Comment by vasugupta9** [comment]
Hi everyone, 👋

My name is Vasu. I heard about this open source community effort on linkedIn and wanted to contribute towards it. My motivation for joining was to contribute to open source, share my knowledge and learn from awesome people here ! I am contributing to Unit 8 on 3D Vision. 

My background is MS in EE from Stanford University (graduated in 2015) and have previously worked in the industry on non-AI (computational electromagnetics, chipset architecture design) and AI (NLP, Computer vision) things ! 

Would love to connect with you all ! You can reach out to me on Discord or my [linkedin](https://www.linkedin.com/in/vasugupta9/)
